cpp/program/playsettings.cpp

```diff
@@ -25,6 +25,10 @@ PlaySettings PlaySettings::loadForMatch(ConfigParser& cfg) {
   playSettings.resignThreshold = cfg.getDouble("resignThreshold",-1.0,0.0); //Threshold on [-1,1], regardless of winLossUtilityFactor
   playSettings.resignConsecTurns = cfg.getInt("resignConsecTurns",1,100);
   playSettings.compensateKomiVisits = cfg.contains("compensateKomiVisits") ? cfg.getInt("compensateKomiVisits",1,10000) : 100;
+  playSettings.policyInitAreaProp = cfg.contains("policyInitAreaProp") ? cfg.getDouble("policyInitAreaProp",0.0,1.0) : 0.04;
```

Comment:
This should not default to 0.04, to avoid changes for people who don't specify these parameters.

Reply:
I just made the default value to 0. In addition, we make the compensateAfterPolicyInitProb as 1 to make sure that the game is fair when `policyInitAreaProp` is set to a value but `compensateAfterPolicyInitProb` is not.

Reply:
Great!